### ISSUE TITLE
Using for cite-seq

### ISSUE THREAD

**Issue by curtisd0886** [sdparekh/zUMIs#257]
Would it be possible to use zUMIs to find feature barcodes in CITE-seq? My though was to make a STAR index file with the CITE-seq sequences and then map with zUMIs. When I have tried this I get an error because all the reads end up in the unmapped bam file. Is there a way to make this work? I think the issue may be with generating an index file with a fasta that contains many small (15nt) sequences.

**Comment by cziegenhain** [comment]
I don't think you can map 15 bp efficiently as a "cDNA" but it would depend on the number of sequences and the settings within star.
Sorry to say we don't currently have a solution for cite-seq, maybe in the future. 
Best, 
Christoph
